dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestCurrentLimitInterface.cpp

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+// This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+// simulation tool for power systems.
+//
+
+#include "DYNCurrentLimitInterfaceIIDM.h"
+
+#include "gtest_dynawo.h"
+
+
+// DG- namespace DYN {
```

Comment:
I agree that it is useless but I would like to keep the same coding style as the others unit tests

Reply:
done